rb/build.desc

```diff
@@ -117,7 +117,8 @@ ruby_library(name = "firefox",
     "lib/selenium/webdriver/firefox.rb"
   ],
   resources = [
-    { "//javascript/webdriver/atoms:getAttribute": "rb/lib/selenium/webdriver/atoms/getAttribute.js"}
+    { "//javascript/webdriver/atoms:getAttribute": "rb/lib/selenium/webdriver/atoms/getAttribute.js"},
+    { "//javascript/firefox-driver:webdriver_prefs" : "rb/lib/selenium/webdriver/firefox/extension/prefs.json" }
```

Comment:
Are the other bindings using things from firefox-driver build to do this? Maybe we can check on how much the prefs.json changes and see if we can just commit a hard coded copy? I don't think we're planning on changing that code much post-geckodriver.